backend/shared_configs/configs.py

```diff
@@ -83,9 +83,15 @@ def validate_cors_origin(origin: str) -> None:
         raise ValueError(f"Invalid CORS origin: '{origin}'")
 
 
-CORS_ALLOWED_ORIGIN = os.environ.get("CORS_ALLOWED_ORIGIN", "*").split(",") or ["*"]
+CORS_ALLOWED_ORIGIN = os.environ.get("CORS_ALLOWED_ORIGIN", "*")
```

Comment:
Let's add some comments here about:

1. Examples of valid values for the env var.
2. What this code is trying to do ... it looks like it's converting the env string to a list and stripping whitespace and/or defaulting to "*"?

Reply:
Generally think that environment variables should be explained in the docs as much as possible rather than in the code that parses them, but I'll add a comment explaining the proper structure